src/client/app/components/visual-unit/VisualUnitDetailComponent.tsx

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { FormattedMessage } from 'react-intl';
+import TooltipHelpComponent from '../TooltipHelpComponent';
+import * as React from 'react';
+import CreateVisualUnitMapModalComponent from './CreateVisualUnitModalMapComponent';
+import CreateCikVisualMapComponent from './CreateCikVisualModalMapComponent';
+import TooltipMarkerComponent from '../TooltipMarkerComponent';
+
+/**
+ * Defines the units and conversion graphics view.
+ * @returns Units visual graphics page element
+ */
+export default function VisualUnitDetailComponent() {
+
+	const titleStyle: React.CSSProperties = {
+		textAlign: 'center'
+	};
+
+	const tooltipStyle = {
+		display: 'inline-block',
+		fontSize: '50%',
+		// For now, only an admin can see the unit page.
+		tooltipVisualUnitView: 'help.admin.unitconversionvisuals'
+	};
+
+	return (
+		<div>
+			<TooltipHelpComponent page='visual-unit' />
+
+			<div className='container-fluid'>
+				<h1 style={titleStyle}>
+					<FormattedMessage id='units.conversion.page.title' />
+					<div style={tooltipStyle}>
+						<TooltipMarkerComponent page='visual-unit' helpTextId={tooltipStyle.tooltipVisualUnitView} />
+					</div>
+				</h1>
+				<br/>
```

Comment:
If you want, CSS styling the component might be a nicer way to get the space (if that is what was desired). I'm also unclear on why this div is different from the one below for what in div.